styleguide/source/_patterns/02-organisms/00-global/01-hub-header/hub-header.twig

```diff
@@ -0,0 +1,13 @@
+<header class="hub-header" role="banner">
+  <div class="hub-header-inner">
+    {% include 'atoms-logo-reversed' %}
+    <div class="hub-header_messaging">
+      {% include 'atoms-h1' with { 'class' : 'hub-header_title', 'content' : 'Become a member today for $35' } %}
+      {% include 'atoms-button-reversed-link' with { 'content' : 'Join Now' } %}
+      <div class="hub-header_small">
+        {% include 'atoms-link-white' with { 'class' : 'hub-header_cta', 'content' : 'Already a Member? Renew.' } %}
+        {% include 'atoms-paragraph' with { 'class' : 'hub-header_text', 'content' : '800-6263211' } %}
```

Comment:
This should account for the telephone number being linked. It could either use the existing link atom or a new telephone link one.